ONSdigital/ras-frontstage
# 269 ext user account unlock

# Motivation and Context
We want to display a template to the respondent when the account is locked and to call party to notify the respondent via gov notify
# What has changed
- Added a party controller to call endpoint in party-service which calls notify gateway
- Added a new template

# How to test?
Run with [Django](https://github.com/ONSdigital/django-oauth2-test/pull/46) and [party](https://github.com/ONSdigital/ras-party/tree/269-ext-user-account-unlock) once account is locked you should see a new template. Then check Party logs to get the url to reset your password and unlock your account.

# Links
[Trello](https://trello.com/c/6IEaOnA9/269-us204-ext-user-account-locked-l)
